lib/matplotlib/backend_bases.py

```diff
@@ -400,11 +401,11 @@ def _iter_collection_uses_per_path(self, paths, all_transforms,
         if Npaths == 0 or (len(facecolors) == 0 and len(edgecolors) == 0):
             return 0
         Npath_ids = max(Npaths, len(all_transforms))
-        N = max(Npath_ids, len(offsets))
+        N = max(Npaths, len(offsets))
         return (N + Npath_ids - 1) // Npath_ids
 
     def _iter_collection(self, gc, master_transform, all_transforms,
-                         path_ids, offsets, offsetTrans, facecolors,
+                         paths, path_ids, offsets, offsetTrans, facecolors,
```

Comment:
@WeatherGod Right, I noticed the new arg at the end, but not the one in the middle.